content/topics/Analyze/machine-learning/ml-intro/machine-learning-101.md

```diff
@@ -0,0 +1,141 @@
+
+Different approaches are **classification** and **regression**: 
+* Classification predicts categorical outcomes, such as labeling students' course feedback as positive or negative. 
+* Regression predicts continuous outcomes, like forecasting sales of a newly launched product.
+
+Popular supervised learning algorithms include linear regression, logistic regression, and decision trees.
+
+### 2. **Unsupervised Learning** 
+<!-- link to unsupervised ml bb -->
+Unsupervised learning finds patterns in data without predefined labels or outcomes. If you want to identify relationships in your data, without a given outcome variable, this is the right approach since it deals with unlabeled data. It explores the data's inherent structure to identify patterns and relationships without explicit guidance or output. Unsupervised learning is often used to group and cluster different objects based on various attributes, such as segmentating customers based on demographics and purchase history.
+
+Different approaches are clustering, generalization and association: 
+* Clustering can group similar instances together, such as organizing articles into topics based on their content.
+* Generalization is used for finding general patterns, like identifying the typical purchasing behavior of different customer segments.
+* Association finds rules that represent large parts of the data, for example, discovering that customers who buy bread often also buy cheese.
+
+Common unsupervised learning techniques include K-means clustering, Principal Component Analysis (PCA), and the Apriori algorithm.
+
+### 3. **Reinforcement Learning** 
+<!-- link to reinforcement ml bb -->
+
+Reinforcement learning teaches a model to make decisions by interacting with its environment. The algorithm receives feedback in the form of rewards or penalties, refining its actions to achieve a specific goal. It is a behavioral modeling technique where the model learns through a trial and error mechanism as it keeps interacting with the environment.
+
+Different approaches are model-based and model-free reinforcement learning:
+* Model-based reinforcement learning predicts the outcomes of actions, such as a navigation system learning the fastest route by simulating different paths based on traffic patterns.
+* Model-free reinforcement learning learns policies directly from experience without prior knowledge of the model of the environment, such as teaching a computer program to play a video game. 
+
+Common reinforcement learning methods are Q-learning and policy gradients.
+
+<!-- If you'd like to learn more about each of these machine learning types, you can check out their corresponding articles, which cover more in-depth theory and examples:
+- Supervised Learning
+- Unsupervised Learning
+- Reinforcement Learning -->
```

Comment:
Were you trying to make a Tip box here? It seems not to render properly. Try to put the text of the box in this format:

{{% tip %}}
your text
{{% /tip %}}

If you want to add links, use markdown format putting the preferred link name between '[ ]' followed by the link between '( )'

Reply:
These were just my notes. This text will refer to the subsequent articles I'll write and once they are published I can link to them with this paragraph